src/main/java/io/citrine/jcc/search/core/result/BaseMultiSearchResult.java

```diff
@@ -55,17 +55,30 @@ public Long getTook() {
     }
 
     /**
+     * Set the list of successes.  This overwrites any successes that are already saved.
+     * @param successes List of successes.
+     * @return This object.
+     */
+    @JsonSetter("successes")
```

Comment:
You are going to hate me for this, but I think we should have a bean that wraps a search result and status. Then we save a list of those objects, one for each result. This way a single object contains the search result and status, rather than having two lists with implied coupling between their elements.